pkg/resources/instance/resource.go

```diff
@@ -34,6 +34,12 @@ func Resource() *schema.Resource {
 			Type:        schema.TypeString,
 			Computed:    true,
 		},
+		AttrDestroyProtected: {
+			Description: "Mark the instance as protected, the Exoscale API will refuse to delete the instance until the protection is removed (boolean; default: `false`).",
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Computed:    true,
```

Comment:
It should not be `Computed` (just `Optional`) as we don't read remote value.

Reply:
Good point. I made it `Computed` because in the long run we plan to implement API calls that allow us to read the remote value. But now I removed it and tested that it can be added later without invalidating an existing and applied tf config. So TIL: adding the `Computed` property to an attribute is a backwards compatible change from the configuration point of view.

However, if you read the comment in my latest commit, you'll see that this brings a slight change in behaviour. But I think this is a good solution.